_template/.azure-pipelines.yml

```diff
@@ -0,0 +1,40 @@
+pool:
+  vmImage: ubuntu-latest
+variables:
+  - name: goVersion
+    value: 1.17
+
+stages:
+  - stage: test
+    jobs:
+      - job: GoBuildTest
+        displayName: Lint & Test Go Application
+        steps:
+          - task: GoTool@0
+            displayName: Install Go Binary
+            inputs:
+              version: $(goVersion)
+
+          - bash: make download
+            displayName: Downloads the dependencies
+
+          - bash: make lint
+            displayName: Lints all code with golangci-lint
+
+          - bash: make test-reports
+            displayName: Runs all tests and generates reports
```

Comment:
If the generated reports are not used we might as well use `make test` since it has more readable output. The output is relevant if you want to upload it somewhere.

Reply:
Good point, I will update it.